eng/build.ps1

```diff
@@ -247,8 +247,8 @@ function BuildSolution() {
   # that MSBuild output as well as ones that custom tasks output.
   $msbuildWarnAsError = if ($warnAsError) { "/warnAsError" } else { "" }
 
-  # Workaround for some machines in the AzDO pool not allowing long paths (%5c is msbuild escaped backslash)
-  $ibcDir = Join-Path $RepoRoot ".o%5c"
+  # Workaround for some machines in the AzDO pool not allowing long paths
```

Comment:
Was the comment applying to the Join-Path itself which is no longer needed?

Reply:
Fixed. I'm iterating on this locally in the VMR. Let me verify it works properly.

Reply:
This works in the VMR now. I think the new comment still applies.